vpn-router-web/src/main/java/vpnrouter/web/validator/NameValidator.java

```diff
@@ -0,0 +1,22 @@
+package vpnrouter.web.validator;
+
+import com.vaadin.flow.data.binder.ValidationResult;
+import com.vaadin.flow.data.binder.Validator;
+import com.vaadin.flow.data.binder.ValueContext;
+import vpnrouter.api.client.ClientConstraints;
+
+public class NameValidator implements Validator<String> {
+    @Override
+    public ValidationResult apply(String value, ValueContext context) {
+        if (value == null || value.isEmpty()) {
+            return ValidationResult.error("Name cannot be empty");
```

Comment:
Why? The name actually can be not given (null)

Reply:
Sorry, I didn't know about this when I wrote the validator. I deleted this validation